services/reddit/subreddit-subscribers.tester.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const { isMetric } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('subreddit-subscribers (valid subreddit)')
+  .get('/drums.json')
+  .expectBadge({
+    label: 'follow r/drums',
+    message: isMetric,
+  })
+
+t.create('subreddit-subscribers (invalid subreddit)')
+  .get('/not-a-real-subreddit.json')
```

Comment:
I'm not entirely sure why, but https://www.reddit.com/r/not-a-real-subreddit/about.json is returning an HTTP 200 response for me, with a response object:

```json
{
  "kind": "Listing",
  "data": {
    "modhash": "",
    "dist": 9,
    "children": [...],
    "after": null,
    "before": null
  }
}
```

This is causing this test to fail in the CI job, so you may want to try changing it to a different value

Reply:
Nice, done. Locally I can't run the tests, and I don't know if I'm doing something wrong.
I type 
`npm run tests:services -- --only=reddit`
and I get
`Error: Unknown services: reddit`

Reply:
you'll want to use the same value you used for the path so try

`npm run tests:services -- --only=subreddit `

Reply:
it looks like that http 200 response is pretty consistent 🤔 I was _finally_ able to get the typical 404 with `abadfbasdfadfadfadfadfasdf` as the subreddit 😄 

This does make me think we may need to tweak the service (schema and `handle` functions) a bit to better handle the subreddit not found scenario. Otherwise users are just going to get the `invalid` or `invalid response data` errors which aren't exactly indicative of the underlying issue in this case

Reply:
Let me research on this then! Thanks for the help!

Reply:
I added several inline suggestions/code blocks that I think should handle this. May be worth giving that approach a try